reference-adapter/mgramseva-ifix-adapter/src/main/resources/application.properties

```diff
@@ -68,7 +68,7 @@ ifix.department.entity.search.path=/departmentEntity/v1/_search
 pspcl.ifix.event.receiver.name=mgramseva-qa
 
 #Value in seconds
-ifix.fiscal.event.search.time.interval.minutes=10
+ifix.fiscal.event.search.time.overlap.minutes=15
```

Comment:
Don't add this in the helm chart. Let this just be here. 
Remove the old variable from the helm chart and env file.

Reply:
will raise PR in DevOp repo